src/bots/pins/subCommands/listPins.js

```diff
@@ -0,0 +1,37 @@
+import client from '../../../client';
+
+// TODO: this should be configurable
```

Comment:
Can you create a low priority issue for this?

Reply:
will do